src/main/java/com/pinterest/secor/parser/TimestampedMessageParser.java

```diff
@@ -26,11 +26,18 @@
 public abstract class TimestampedMessageParser extends MessageParser {
 
     private SimpleDateFormat mFormatter;
+    Object outputPattern = null;
```

Comment:
Think this ought to be a `String`, but according to https://github.com/pinterest/secor/pull/10#discussion_r15147319 I think the style in this project is to just access the config inline as needed.